README.md

```diff
@@ -154,7 +159,8 @@ Filters format:
             "operation": "WILDCARD"
           }
         ]
-      }
+      },
+      "virtualQueueLimit": 10000
```

Comment:
I'm not sure that this option is required.

Reply:
Renamed to `virtualPublishLimit` and moved to `mq_router.json` as we decided.